gqlalchemy/changelog.md

```diff
@@ -6,18 +6,42 @@ sidebar_label: Changelog
 
 ## v1.2 - Apr 12, 2022
 
+:::warning
+
+### Breaking Changes
+
+- Ordering query results as in GQLAlchemy older than 1.2 will not be possible.
+- `where()`, `and_where()` and `or_where()` methods can't be used as in
+  GQLAlchemy older than 1.2.
+- Setting up the `bootstrap_servers` argument when creating a stream as in
+  GQLAlchemy older than 1.2 will not be possible.
+
+:::
+
 ### Major Features and Improvements
 
-- Fixed bugs related to contraints check.
-- Fixed label inheritance.
-- Fixed `save_node()` method.
-- Improved `where()` method in query builder.
-- Added batch save methods: `save_nodes()` and `save_relationships()`.
-- Added `drop_triggers` method for dropping all triggers.
-- Added new methods to the query builder: `load_csv()` and `xor_where()`.
-- Added support for creating a trigger without `ON`.
-- Added Memgraph instance runner.
-- Added Azure Blob importer.
+- Added `set_()` method to query builder. [#128](https://github.com/memgraph/gqlalchemy/pull/128)
+- Improved `where()`, `and_where()`, `or_where()` and `xor_where()` methods. [#114](https://github.com/memgraph/gqlalchemy/pull/114)
+- Added `where_not()`, `and_not()`, `or_not()` and `xor_not()` methods. [#114](https://github.com/memgraph/gqlalchemy/pull/114)
+- Improved `order_by()` method from query builder by changing its argument types. [#114](https://github.com/memgraph/gqlalchemy/pull/114)
+- Added Docker and Binary Memgraph instance runners. [#91](https://github.com/memgraph/gqlalchemy/pull/91)
+- Added Azure Blob and local storage importers. [#104](https://github.com/memgraph/gqlalchemy/pull/104)
```

Comment:
maybe put this after "Added table to graph importer", which, also, added the S3 importer, so we can note that somewhere